packages/blocks/src/api/registration.js

```diff
@@ -199,6 +199,15 @@ export function getFreeformContentHandlerName() {
 	return select( 'core/blocks' ).getFreeformFallbackBlockName();
 }
 
+/**
+ * Retrieves name of block used for handling grouping interactions
```

Comment:
Didn't we have ESLint rule which enforces period at the end of the sentence?
```suggestion
 * Retrieves name of block used for handling grouping interactions.
```

Reply:
Not sure. Fixed anyway.